src/test/java/ome/services/blitz/test/utests/ServiceFactoryServiceCreationDestructionTest.java

```diff
@@ -113,14 +111,12 @@ public Object invoke(MethodInvocation arg0) throws Throwable {
     @Test
     public void testDoStatelessAddsServantToServantListCacheAndAdapter()
             throws Exception {
-        IAdminPrxHelper admin = IAdminPrxHelperFactory.create();
 
         callsActiveServices(Collections.singletonList(adminServiceId));
-        map.put(adminServiceId, new _IAdminTie());
         mockAdapter.expects(once()).method("add").will(returnValue(null));
         mockAdapter.expects(once()).method("find").will(returnValue(null));
         mockAdapter.expects(once()).method("createDirectProxy").will(
-                returnValue(admin));
+                returnValue(null));
```

Comment:
That's an interesting change.

Reply:
This is one the ``blitz->server`` relationship

Reply:
Sorry wrong repo

Reply:
It matches the other tests

Reply:
Aha, thanks.